### PR TITLE
first stab at v0.1.0 of standard terraform module

### PR DESCRIPTION
**Pull request by yoshutch**
This is my first stab at the overarching pattern module that creates a fargate service and all other resources needed to get up and running.

This doesn't offer much configuration yet, so I'm open for suggestions on what we need to allow to be configured in this standard module.

### CODE REVIEW

File: examples/simple/example.tf [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+module "fargate_api" {
+  source = "git@github.com:byu-oit/terraform-aws-standard-fargate?ref=v1.0.0"
+//  source     = "../../" // for local testing
+  app_name   = "example-api"
+  image      = "${data.aws_ecr_repository.pre_made_repo.repository_url}:test"
+  image_port = 8000
+}
+
+output "appspec" {
+  value = module.fargate_api.appspec
```
Comment (jvisker) [inline examples/simple/example.tf]: What is "appspec"? Should it be "codedeploy_appspec_json"?

Comment (yoshutch) [reply examples/simple/example.tf]: Done

File: README.md [hunk 2]
```diff
@@ -1,27 +1,63 @@
...
 ## Inputs
 | Name | Description | Default |
 | --- | --- | --- |
-| | |
+| app_name | Application name to name your Fargate API and other resources | |
+| image | The docker image (including tag). Include the full URI if not pulling from docker hub | |
+| image_port | The port the docker image is listening on | |
+| health_check_path | Health check path for the image | "/" |
+| container_env_variables | Map of environment variables to pass to the container definition | {} |
+| container_secrets | Map of secrets from the parameter store to be assigned to env variables. Use `task_policies` to make sure the Task's IAM role has access to the SSM parameters | {} |
```
Comment (joshgubler) [inline README.md:51]: Where is `task_policies` exposed?

Comment (yoshutch) [reply README.md:51]: It wasn't. Now it should be done.

**Review by joshgubler — APPROVED** [review]

**Comment by joshgubler** [comment]
This looks like a great start. I'm sure that as configuration needs arise, we can add inputs as necessary.

**Pull request merged**